lib/src/lib.rs

```diff
@@ -52,6 +52,7 @@ pub mod fmt_util;
 pub mod fsmonitor;
 #[cfg(feature = "git")]
 pub mod git;
+pub mod trailer;
```

Comment:
nit: move it after `pub mod time_util;`

Reply:
This is marked resolved but it doesn't seem to be fixed. Did you forget to push after fixing it?